src/System.Text.Encoding.CodePages/src/System/Text/EncodingTable.Data.cs

```diff
@@ -1,22 +1,21 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 //
-// THIS IS AUTOGENERATED FILE
-// USE ENCODINGDATA TOOL TO GENERATE SUCH FILE
+// THIS IS AN AUTOGENERATED FILE
+// IT IS GENERATED FROM A BUILDTOOLS TASK
```

Comment:
Yes I mean it will be nice the comment here explain more how this file get generated. could you please create a new PR there and I'll merge it. sorry I looked at these changed after I have merged the other one.